packages/pigeon/example/app/macos/Runner/messages.g.h

```diff
@@ -56,11 +56,17 @@ NSObject<FlutterMessageCodec> *PGNExampleHostApiGetCodec(void);
 extern void SetUpPGNExampleHostApi(id<FlutterBinaryMessenger> binaryMessenger,
                                    NSObject<PGNExampleHostApi> *_Nullable api);
 
+extern void SetUpPGNExampleHostApiWithSuffix(id<FlutterBinaryMessenger> binaryMessenger,
+                                             NSObject<PGNExampleHostApi> *_Nullable api,
+                                             NSString *messageChannelSuffix);
+
 /// The codec used by PGNMessageFlutterApi.
 NSObject<FlutterMessageCodec> *PGNMessageFlutterApiGetCodec(void);
 
 @interface PGNMessageFlutterApi : NSObject
 - (instancetype)initWithBinaryMessenger:(id<FlutterBinaryMessenger>)binaryMessenger;
+- (instancetype)initWithBinaryMessenger:(id<FlutterBinaryMessenger>)binaryMessenger
+                   messageChannelSuffix:(nullable NSString *)messageChannelSuffix;
```

Comment:
is it intended for this to be nullable (while swift version is nonnull)?

Reply:
becuase @stuartmorgan told me to. I can make the swift version nullable too, if you think that's more "swift style" than the current implementation.

Reply:
In general I don't know that it needs to be nullable, seems weird to have a nullable argument when there's already a version of the method that doesn't have that parameter at all.

Reply:
it is nullable in swift anyway isn't it? it has a default value at least

Reply:
it's better to have non-null in swift, so that you have 1 less case to deal with. 

Just wanted point out the fact that the divergence of the 2 APIs.

Reply:
> In general I don't know that it needs to be nullable, seems weird to have a nullable argument when there's already a version of the method that doesn't have that parameter at all.

It's not that it needs to be, it's that the standard practice for Obj-C APIs (e.g., what you'll see in Foundation APIs) is that if you have multiple variants of methods, the extras are generally nullable, indicating that the shorter methods are equivalent to calling the longer ones with nulls.

If it makes life harder for it to be nullable, you could make it non-nullable, I just think it'll just feel somewhat odd to Obj-C developers.

Reply:
It's not any harder for it to be nullable, the logic is basically the same. I think it's good as is.